### PR TITLE
fix: prevent immediate run of wallet recovery on cron script

### PR DESCRIPTION
**Pull request by sdbondi**
Description
---
When running cron, the two wallet recovery tests would run immediately
and not be awaited (Promises are eager in js)

Motivation and Context
---
Minor fix

How Has This Been Tested?
---
Script run and tests did not immediately start

### CODE REVIEW

File: applications/daily_tests/automatic_recovery_test.js [hunk 1]
```diff
@@ -84,14 +84,14 @@ async function run(options = {}) {
         let successLog = data.match(RECOVERY_COMPLETE_REGEXP);
         let recoveredAmount = data.match(RECOVERY_WORTH_REGEXP);
         if (successLog && recoveredAmount) {
-          let recoveredAmount = parseInt(recoveredAmount[1]);
+          let recoveredAmt = parseInt(recoveredAmount[1]);
```
Comment (SWvheerden) [inline applications/daily_tests/automatic_recovery_test.js]: nit: I prefer the full typed out name, it's more readable than Amt.

**Review by SWvheerden — DISMISSED** [review]
Looks good to me

File: applications/daily_tests/automatic_recovery_test.js (hunk 1 above)
Comment (sdbondi) [reply applications/daily_tests/automatic_recovery_test.js]: It's already used - you cannot rebind variables in js. I will change the variable names to be better

Comment (delta1) [reply applications/daily_tests/automatic_recovery_test.js]: `let` is block scoped so you should be able to reuse the name if you really want to 

https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Statements/let

**Review by stringhandler — DISMISSED** [review]

File: applications/daily_tests/automatic_recovery_test.js (hunk 1 above)
Comment (sdbondi) [reply applications/daily_tests/automatic_recovery_test.js]: yeah, but both var names are in this scope :P

Comment (sdbondi) [reply applications/daily_tests/automatic_recovery_test.js]: Changed it in any case to `recoveredAmountMatch` and `recoveredAmount`

**Review by stringhandler — APPROVED** [review]

**Pull request merged**